renovate-config.json

```
@@ -0,0 +1,40 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "timezone": "Europe/Warsaw",
+  "extends": ["config:base", ":disableDependencyDashboard"],
```

```suggestion
  "extends": ["config:base", ":disableDependencyDashboard", ":semanticCommitTypeAll(chore)"],
  "npm": {
    "commitMessageTopic": "{{prettyDepType}} {{depName}}",
    "commitMessagePrefix": "chore(deps): :arrow_up: "
  },
```

Based on my observation of our repositories, it appears that we adhere to semantic commit standards. Therefore, it would be beneficial to implement them here as well.

---

Renovate adheres to conventional commits when using `config:base`: [docs](https://docs.renovatebot.com/semantic-commits/#semantic-commit-messages), [example](https://github.com/vuestorefront/storefront-ui/pull/2735)

---

Yes, but it 
> uses the fix prefix for npm production dependencies

I just not sure is that ok - I personally prefer to use `chore` for all upgrades, but I'm not insisting on it ;)

---

I'd say that it depends on the scope of the update - sometimes when we do an upgrade which contains breaking changes of apis we expose it could probably be even a major release 😄
But I also have no hard opinion about this one, let's change it then